docs/src/getting-started/scarb.md

```diff
@@ -10,5 +10,14 @@ Starknet Foundry uses [Scarb](https://docs.swmansion.com/scarb) to:
 One of the core concepts of Scarb is its [manifest file](https://docs.swmansion.com/scarb/docs/reference/manifest.html) - `Scarb.toml`.
 It can be also used to provide [configuration](../projects/configuration.md) for Starknet Foundry.
 
+Starknet Foundry requires your project to be compiled in Sierra and also CASM. Scarb allows you to do so using it's target system.
+Consider adding the respective codegen parameters in the `[[target.starknet-contract]]` table of your `Scarb.toml` file.
+
+```toml
+[[target.starknet-contract]]
+sierra = true
+casm = true
+```
+
```

Comment:
Please remove this change - this page is a general overview what Scarb is and what it is used for, it shouldn't contain any specific information like that. I think the error message + info in first steps is sufficient